src/components/RadioButton.js/RadioButton.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import RadioButton from '.';
+
+
+export default {
+  title: 'RadioButton',
+  component: 'RadioButton'
+};
+
+
+const onChange = () => console.log('Value has changed');
+
+
+   export const defaultRadioButton = () =>  <RadioButton  />
```

Comment:
Nice work on this but some of code doesn't seem well formatted, you can run `npm run fix:lint` to fix most of them